patches/v8/0003-Allow-Windows-builds-under-Bazel.patch

```diff
@@ -1,113 +1,9 @@
-             "src/base/debug/stack_trace_win.cc",
-             "src/base/platform/platform-win32.cc",
-+            "src/base/platform/platform-win32.h",
-             "src/base/win32-headers.h",
-         ],
-     }),
-@@ -1271,6 +1272,7 @@ filegroup(
-         "include/v8-wasm-trap-handler-posix.h",
-         "src/api/api.cc",
-         "src/api/api.h",
-+        "include/v8-wasm-trap-handler-win.h",
-         "src/api/api-arguments.cc",
-         "src/api/api-arguments.h",
-         "src/api/api-arguments-inl.h",
-@@ -2757,6 +2759,11 @@ filegroup(
-             "src/trap-handler/handler-inside-posix.cc",
-             "src/trap-handler/handler-outside-posix.cc",
-         ],
-+        "@v8//bazel/config:is_windows": [
-+            "src/trap-handler/handler-inside-win.cc",
-+            "src/trap-handler/handler-inside-win.h",
-+            "src/trap-handler/handler-outside-win.cc",
-+        ],
-         "//conditions:default": [],
-     }) + select({
-         "@v8//bazel/config:v8_arm64_simulator": [
-@@ -2764,13 +2771,6 @@ filegroup(
-             "src/trap-handler/trap-handler-simulator.h",
-         ],
-         "//conditions:default": [],
--    }) + select({
--        "@v8//bazel/config:is_windows": [
--            "src/trap-handler/handler-inside-win.cc",
--            "src/trap-handler/handler-inside-win.h",
--            "src/trap-handler/handler-outside-win.cc",
--        ],
--        "//conditions:default": [],
-     }) + select({
-         "@v8//bazel/config:is_windows_64bit": [
-             "src/diagnostics/unwinding-info-win64.cc",
-@@ -3833,6 +3833,9 @@ filegroup(
-         "@v8//bazel/config:is_msvc_asm_ia32": ["src/heap/base/asm/ia32/push_registers_masm.asm"],
-         "@v8//bazel/config:is_msvc_asm_x64": ["src/heap/base/asm/x64/push_registers_masm.asm"],
-         "@v8//bazel/config:is_msvc_asm_arm64": ["src/heap/base/asm/arm64/push_registers_masm.S"],
-+        "@v8//bazel/config:is_windows_clang_asm_ia32": ["src/heap/base/asm/ia32/push_registers_asm.cc"],
-+        "@v8//bazel/config:is_windows_clang_asm_x64": ["src/heap/base/asm/x64/push_registers_asm.cc"],
-+        "@v8//bazel/config:is_windows_clang_asm_arm64": ["src/heap/base/asm/arm64/push_registers_asm.cc"],
-     }),
- )
- 
-@@ -4227,9 +4230,11 @@ filegroup(
-         "src/d8/d8-js.cc",
-         "src/d8/d8-platforms.cc",
-         "src/d8/d8-platforms.h",
--        "src/d8/d8-posix.cc",
-         "src/d8/d8-test.cc",
--    ],
-+    ] + select({
-+        "@v8//bazel/config:is_posix": ["src/d8/d8-posix.cc"],
-+        "@v8//bazel/config:is_windows": ["src/d8/d8-windows.cc"],
-+    }),
- )
- 
- genrule(
-@@ -4581,7 +4586,7 @@ py_test(
-         ":noicu/d8",
-         ":noicu/v8_build_config",
-         "//testing/pybase",
--    ] + glob(["test/**"]) + glob(["tools/**/*.js"]) + glob(["tools/**/*.mjs"]),
-+    ] + glob(["test/**"], exclude = ["test/message/unicode-filename-*"],) + glob(["tools/**/*.js"]) + glob(["tools/**/*.mjs"]),
```

Comment:
Not sure if this patch is still needed – it hasn't been ported over, we'll see if Windows still has problems with the `test/message/unicode-filename-🎅🎄.js` filename.